types/pull-stream/index.d.ts

```diff
@@ -1,10 +1,22 @@
 // Type definitions for pull-stream 3.6
 // Project: https://pull-stream.github.io
-// Definitions by: Michael de Wit <https://github.com/mjwwit>
+// Definitions by: Rong Shen <https://github.com/jacobbubu>
```

Comment:
Is this common practice? Changing the author of the definitions? I'm asking because I have no idea...

Reply:
I'm not sure, just following @andrewbranch's advice.

https://github.com/DefinitelyTyped/DefinitelyTyped/pull/46261#issuecomment-674160905

Reply:
Ah right, I must've missed that.